tools/dependency-check/known-dependencies.txt

```diff
@@ -25,6 +25,7 @@ cache-api-1.1.1.jar
 checker-qual-3.12.0.jar
 cloudevents-api-2.4.2.jar
 cloudevents-core-2.4.2.jar
+cloudevents-http-vertx-2.3.0.jar
```

Comment:
Many newly added dependencies are indirectly imported by vertx. I can't pass the `License Check` of CI/CD after removing them. 

Actually, I'm confused about `known-dependencies.txt`. Why do we need to manually add dependencies here? Since we are already using a build tool.

Reply:
We need to use the known-dependencies to recognize which packages are new dependencies and make sure these new dependencies must under the ASF 3rd party license policy. @Fungx @pandaapo